### PR TITLE
feat: index cloud cognitive

### LINKED ISSUE #731: Index IBM Products
Name: IBM Products

Description: A curated set of components and patterns, built on top of Carbon and designed by the Carbon for IBM Products team.

Repo: https://github.com/carbon-design-system/ibm-cloud-cognitive/tree/main/packages/cloud-cognitive

Storybook: https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products

We don't need to index assets tagged as "Legacy". No need to index library `navData` or asset `docs` yet as we want to prove that with a Carbon core-maintained library first.

- [ ] Platform squad to open initial indexing PR in remote repo using the one-carbon.yml-per-library indexing approach
- [ ] Identify and share gaps with David, Elicia, and Cameron
- [ ] Open draft PR in this repo to register the library
- [ ] Once gaps are filled (e.g. all assets indexed with demo links, tags, status, thumbnail, etc.) and once the remote PR is merged, merge the platform draft PR

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #731 

Repo PR: https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/2063

Filter by:  `IBM Products`

- [x] Missing thumbnail `expressive-card`

## TESTING

- 38 total indexed
- 22 patterns
Screen Shot 2022-07-05 at 4 49 16 PM

- 16 components
Screen Shot 2022-07-07 at 10 14 57 AM

TODOS: 

- [x] @aubrey-oneal double check tags
- [x] @jeanservaas thumbnails
- [x] @andreancardona double check repo details 
- [x] commit cached files  (after ref change)
- [x] switch org, remove ref

### CODE REVIEW

**Review by francinelucca — COMMENTED** [review]
confirmed all components and patterns render 👯 ✨ , added some CRs on repo pr

**Review by francinelucca — COMMENTED** [review]
LGTM! 💃

**Review by francinelucca — COMMENTED** [review]
Does http://localhost:3000/assets/components render for you? I get an error 🤔 . Should this branch pull from main? looks like it's using the old urls, not sure if that's the issue.
image

Also, let's remove these cached files since they come from the andreancardona org and re-add them once we change the ref

**Comment by andreancardona** [comment]
> LGTM except for expressive-card thumbnail. I also only see 14 components, not 16, looks like options-tile is being skipped (I think 'cause the cached file had an error), are we sure it should be 16, not 15? I counted 37 indexed assets 🤔 image

should only be 15 sorry about that!!

File: services/web-app/data/libraries.mjs [hunk 1]
```diff
@@ -119,6 +119,13 @@ const prodLibraries = {
     path: '/',
     group: carbonComponentsGroup
   },
+  'ibm-cloud-cognitive': {
+    host: 'github.com',
+    org: 'carbon-design-system',
+    repo: 'ibm-cloud-cognitive',
+    path: '/packages/cloud-cognitive',
+    sponsor: 'ibm-products'
```
Comment (alisonjoseph) [inline services/web-app/data/libraries.mjs:127]:
```suggestion
    maintainer: 'ibm-products'
```

**Pull request merged**